### DISCUSSION TITLE
Request failed with OR_BIBED_13 error

### DISCUSSION THREAD

**Discussion by bortamas** [flutter-stripe/flutter_stripe#1311] [Q&A] [answered]
**Describe the bug**

**To Reproduce**
1. I'm trying to use Stripe.instance.confirmPlatformPayPaymentIntent() function for payment
2. Everything is set up (app on pay console is approved)
3. When I call confirmPlatformPayPaymentIntent() with "testEnv = false" I get this error:
![IMG_0002](https://github.com/flutter-stripe/flutter_stripe/assets/106537590/960f050a-b895-47d9-ba07-0c2e0fb4b3cd)

**Expected behavior**
Google Pay bottom sheet should be displayed

**Smartphone / tablet**
 - Device: [Oneplus 8T]
 - OS: [Android 13]
 - Package version: [9.2.2 ]
 - Flutter version [3.7.12]

**Additional context**
With "testEnv = true" it works as expected. 
I couldn't find any thread with OR_BIBED_13 error code.

**Comment by hateshkumar** [comment]
Hey, can you please tell me if you have successfully generated the token? how?

**Comment by bortamas** [comment]
@hateshkumar Hi Kumar. Which token are you referring to?

**Comment by hateshkumar** [comment]
> @hateshkumar Hi Kumar. Which token are you referring to?

can you please take a look into this issue you will probably get an idea of what I am referring to. I am stuck badly.
#1300

**Comment by bortamas** [comment]
> > @hateshkumar Hi Kumar. Which token are you referring to?
> 
> can you please take a look into this issue you will probably get an idea of what I am referring to. I am stuck badly. #1300

> 

I am not using that method. I get a client secret from my backend and I'm using the `confirmPlatformPayPaymentIntent({
    required String clientSecret,
    required PlatformPayConfirmParams confirmParams,
  })` function to show the payment sheets.

**Comment by extmkv** [comment]
@bortamas is this app signed with prod key or with firebase distribution one?

**Comment by remonh87** [comment]
To be clear this issue is coming from the Stripe native sdk and it looks like something is configured incorrectly. I would recommend checking your payment  configuration. Also keep in mind in order for google pay to work in production you need to do extra things as stated here: https://stripe.com/docs/google-pay?platform=react-native#going-live-react-native

**Comment by bortamas** [comment, marked as answer]
Thanks for all the help. Turned out it only works if I created a play store internal testing build.
